src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_x.h

```diff
@@ -253,7 +253,7 @@ namespace sirf {
 			return "ImagesReconstructor";
 		}
 
-		void process(MRAcquisitionData& acquisitions);
+		void process(MRAcquisitionData& acquisitions, const char* dcm_prefix = "image");
```

Comment:
I'd prefer to have a `set_dcm_prefix` or similar function. Adding it as an optional argument is rather non-standard.

Reply:
done